executor/brie.go

```diff
@@ -574,3 +574,13 @@ func (gs *tidbGlueSession) UseOneShotSession(store kv.Storage, closeDomain bool,
 	// in SQL backup. we don't need to close domain.
 	return fn(gs)
 }
+
+// parseStorage parses the storage URL
+func parseStorage(storage string) (*url.URL, error) {
```

Comment:
can we use `ParseRawURL` directly?

Reply:
Done. PTAL

Reply:
I moved the unit test into `parse_test.go`.